apm-configuration/node-js.mdx

```diff
@@ -96,6 +96,18 @@ MW_AGENT_SERVICE=mw-service.mw-agent-ns.svc.cluster.local
             
 Distributed tracing is automatically enabled upon completion of Step 2.
 
+#### Span Attributes
+
+For any automatically instrumented spans or endpoints, add the following code snippet:
```

Comment:
I would just put endpoints. Because you don't instrument spans. Spans are a result of instrumentation